### PR TITLE
File Added : Bootstrap Assignment 3 & 4

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

**Review by vijaya1811 — APPROVED** [review]

File: urmila/BOOTSTRAP/Assignment3/Q1.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                                minima delectus tempore veniam eveniet assumenda voluptate maxime facilis ut, doloribus
+                                magnam voluptatibus consequuntur quod?</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/BOOTSTRAP/Assignment3/Q1.html:83]: create proper indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: urmila/BOOTSTRAP/Assignment3/Q1.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (shubhamdalal49) [inline urmila/BOOTSTRAP/Assignment3/Q1.html]: check indentation error

File: urmila/BOOTSTRAP/Assignment4/Q1.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+        <title>bootstrap-ass4</title>
+        <style>
+          img {
+              opacity: 0.5;
+              }
+          img:hover {
+                    opacity: 1.0;
+                  }
```
Comment (shubhamdalal49) [inline urmila/BOOTSTRAP/Assignment4/Q1.html]: give indentation properly

File: urmila/BOOTSTRAP/Assignment4/Q1.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      </style>
+    </head>
+    <body>
+        <div class="jumbotron" style="background-image: url(1.jpg); background-attachment:  no-repeat center fixed; background-size: cover; ">
+                <p class="lead  text-center">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Totam, sint.</p><br>
+                <center>
+                    <a name="" id="" class="btn btn-primary btn-center" href="#" role="button">Click</a>
+                    <a name="" id="" class="btn btn-dark" href="#" role="button">Click</a>
+                </center>             
+        </div>
```
Comment (shubhamdalal49) [inline urmila/BOOTSTRAP/Assignment4/Q1.html]: check indentation and do it properly

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Pull request merged**